mmil.raw.html

```diff
@@ -12302,6 +12306,419 @@
+    <td>The [Geuvers] reference describes a proof
+    without excluded middle, and presumably
+    could be used as our guide.</td>
+  </tr>
+
+  <tr>
+    <td>4.  Pythagorean Theorem</td>
+    <td>There are a variety of theorems
+    which might plausibly deserve the label of
+    "Pythagorean Theorem" but assuming we stick
+    with pythi from set.mm, there are a number of
+    prerequisites.</td>
+  </tr>
+
+  <tr>
+    <td>5.  The Prime Number Theorem</td>
+    <td>The set.mm proof uses df-rlim .</td>
+  </tr>
+
+  <tr>
+    <td>7.  Law of Quadratic Reciprocity</td>
+    <td>Needs more development of number theory (for one
+    thing, the Legendre symbol, df-lgs , itself).</td>
+  </tr>
+
+  <tr>
+    <td>9.  The Area of a Circle</td>
+    <td>At first glance defining Lebesgue measure would
+    not proceed quite as in set.mm, as it is in terms of
+    an infimum.</td>
+  </tr>
+
+  <tr>
+    <td>11.  The Infinitude of Primes</td>
+    <td>Apparently would not be difficult. Note that
+    set.mm has infpn , infpn2 , prminf , and perhaps other
+    forms. Some of them are connected by unben , which
+    won't work as-is (see ~ exmidunben ), but since
+    primality is decidable it should be possible to
+    define something like unben for decidable subsets.</td>
+  </tr>
+
+  <tr>
+    <td>14.  Summation of ` sum_ k e. NN ( k ^ -u 2 ) `</td>
+    <td>Needs a closer look at the set.mm proof but we
+    do have ~ climsqz for whatever that is worth</td>
+  </tr>
+
+  <tr>
+    <td>15.  The Fundamental Theorem of Integral Calculus</td>
+    <td>Although iset.mm has at least a start on developing
+    derivatives, it has nothing on integrals yet</td>
+  </tr>
+
+  <tr>
+    <td>18.  Liouville's Theorem and the Construction
+of Transcendental Numbers</td>
+    <td>Need to develop polynomials (and figure out how to
+    appropriately state the theorem, as the set.mm statement
+    seems to rely on real number equality in a way which
+    may not apply).</td>
+  </tr>
+
+  <tr>
+    <td>19.  Four Squares Theorem</td>
+    <td>The proof is long and needs, at least,
+    Gaussian integers ( df-gz ).</td>
+  </tr>
+
+  <tr>
+    <td>20.  All Primes (1 mod 4) Equal the Sum of Two Squares</td>
+    <td>At first glance it would appear this would intuitionize
+    without much trouble.</td>
+  </tr>
+
+  <tr>
+    <td>22.  The Non-Denumerability of the Continuum</td>
+    <td>See the <a href="#missing-ruc">ruc entry above</a>.
+    We should be able to prove
+    this given ` CCHOICE ` .  As for showing that it cannot
+    be done given just the iset.mm axioms, that is harder,
+    and the proof in [BauerHanson] uses something called
+    parameterized realizability which is a different technique
+    than proving constructive taboos as we have done elsewhere
+    in iset.mm.</td>
+  </tr>
+
+  <tr>
+    <td>23.  Formula for Pythagorean Triples</td>
+    <td>This is a long proof but at first glance should
+    work in iset.mm.</td>
```

Comment:
Incidentally I was thinking of converting this and lemmas to deduction form so, maybe don't work on this yet

Reply:
I saw at least one use of `dedth` and often theorems/steps in inference form need to be rewritten in deduction (or closed) form to intuitionize it.

As for Pythagorean Triples, deduction form does seem like it could be an improvement, but it isn't necessarily a blocker for intuitionizing, assuming the lemmas are now in closed form.